bigbluebutton-tests/playwright/README.md

```
@@ -1,61 +1,94 @@
 ```bash
 npm run test:headed chat
 ```
 
-If you want to run a specific test or a specific group of tests, you can do so with the following command:
+- If you want to run a specific test or a specific group of tests, you can do so with the following command:
 
 ```bash
 npm run test:filter "Send public message"
 ```
 
 _Note: This filter needs to be passed in double quotes_
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)